urbackupserver/www2/src/api/urbackupserver.ts

```diff
@@ -1100,17 +1106,7 @@ class UrBackupServer {
     const salt=randomString();	
 	  const password_md5=MD5(salt+password).toString();
     const params: Record<string, string> = { sa: "useradd", name: name, pwmd5: password_md5, salt: salt };
-    let i = 0;
-    let idx = "";
-    for (const right of rights) {
-      params[i+"_domain"] = right.domain;
-      params[i+"_right"] = right.right;
-      i++;
-      if(idx.length>0)
-        idx += ",";
-      idx += "" + i;
-    }
-    params["idx"] = idx;
+    params['rights'] = formatUserRights(rights)   
```

Comment:
Wouldn't this URLEncode all the parameters into the one 'rights' parameter (which is not how the server expects it)